src/Microsoft.Data.SqlClient/tests/ManualTests/TracingTests/DiagnosticTest.cs

```diff
@@ -198,18 +193,18 @@ public void ExecuteReaderWithCommandBehaviorTest()
 
                         conn.Open();
                         SqlDataReader reader = cmd.ExecuteReader(CommandBehavior.Default);
-                        while (reader.Read()) { }
+                        while (reader.Read())
+                        { }
                     }
                 });
-                return SuccessExitCode;
+                return RemoteExecutor.SuccessExitCode;
             }).Dispose();
         }
 
-        [ConditionalFact(nameof(IsConnectionStringConfigured))]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.NetFramework,  "Internals reflection not supported on UapAot | Feature not available on Framework")]
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
         public void ExecuteXmlReaderTest()
         {
-            RemoteInvoke(cs =>
+            RemoteExecutor.Invoke(cs =>
             {
                 CollectStatisticsDiagnostics(_ =>
```

Comment:
Though this is not new change, what is this `_` for?

Reply:
This is discard. Starting from C# 7 we have that. you can read more about it [here]()https://docs.microsoft.com/en-us/dotnet/csharp/discards